cargo-powdr/template/Cargo.toml.template

```diff
@@ -8,7 +8,7 @@ default = []
 simd = ["powdr/plonky3-simd"]
 
 [dependencies]
-powdr = { git = "https://github.com/powdr-labs/powdr", tag = "v0.1.1", features = ["plonky3"] }
+powdr = { git = "https://github.com/powdr-labs/powdr", tag = "v0.1.2", features = ["plonky3"] }
```

Comment:
```suggestion
powdr = { git = "https://github.com/powdr-labs/powdr", tag = "v0.1.3", features = ["plonky3"] }
```

Why `1.0.2`?

Reply:
the template must point to an existing version, the latest release is 0.1.2.
The 0.1.3 updates are for the dev version (current main)